src/nvim/vim.h

```diff
@@ -71,6 +71,7 @@ enum { NUMBUFLEN = 65, };
 #define SELECTMODE      0x1000  // Select mode, only for mappings
 #define TERM_FOCUS      0x2000  // Terminal focus mode
 #define CMDPREVIEW      0x4000  // Showing 'inccommand' command "live" preview.
+#define PREVIEW_EXEC    0x8000  // Executing the command for 'inccommand' command "live" preview
```

Comment:
oh dear. we're adding another special state? can this be avoided?

Reply:
> oh dear. we're adding another special state? can this be avoided?

I guess we could use a global variable

Reply:
hoping to avoid any kind of new global flag. It's really a headache that we have CMDPREVIEW and now we need _yet another_ global special state _inside_ that already special global state.